src/components/Icon/Icon.js

```diff
@@ -29,12 +34,23 @@ const Icon = (props) => {
   let componentStyle = Object.assign({}, defaultStyle)
   // assign new values from any styles passed as props
   Object.assign(componentStyle, props.style)
-  // handle sizes
-  if (props.small) {
+  // handle width
+  if (props.xsmall) {
```

Comment:
How about?

`size: React.PropTypes.oneOf(['xsmall', 'small', 'large']),`

Then we can do:
```
if (props.size) {
  Object.assign(componentStyle, styles[props.size])
}
```

We would need to also define all the sizes in an style object.

What do you think?